label_studio/data_manager/functions.py

```diff
@@ -314,16 +314,19 @@ def get_prepared_queryset(request, project):
     return queryset
 
 
-def evaluate_predictions(tasks):
-    """Call ML backend for prediction evaluation of the task queryset"""
+def retrieve_predictions(tasks, backend=None):
+    """Call ML backend to retrieve predictions with the task queryset as an input"""
     if not tasks:
         return
 
-    project = tasks[0].project
+    if not backend:
+        project = tasks[0].project
+        backend = project.ml_backends.first()
```

Comment:
this is a small table so it likely doesn't matter, but it's generally better to use `fast_first` if you don't care about ordering

Reply:
since it is only one backend connected all the time, I wouldn't introduce this change rn to avoid big diff